pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -140,11 +155,13 @@ type OperatorConfigurationData struct {
 	Scalyr                     ScalyrConfiguration          `json:"scalyr"`
 }
 
+// OperatorConfigurationUsers defines configration for super user
 type OperatorConfigurationUsers struct {
 	SuperUserName            string            `json:"superuser_name,omitempty"`
 	Replication              string            `json:"replication_user_name,omitempty"`
 	ProtectedRoles           []string          `json:"protected_roles,omitempty"`
 	TeamAPIRoleConfiguration map[string]string `json:"team_api_role_configuration,omitempty"`
 }
 
+//Duration is time.Duration
```

Comment:
I think if we want to avoid golint warnings, we can use this opportunity to do better commentaries and explain "why" instead of "what". E.g. here the purpose of this type alias is probably just to shorten a bit this frequently used name.